docs/index.md

```diff
@@ -18,7 +18,7 @@ research problems and Department of Energy mission needs while efficiently using
 ## Component Models
 
 - [EAM](./EAM/index.md)
-- EAMxx — not yet supported.
+- [EAMxx](https://docs.e3sm.org/scream/)
```

Comment:
keep the "not yet supported" text

Reply:
agreed. let's keep the not yet supported. Would this PR be the appropriate place to fix the typo "Pleaae" or something like that in the banner we have at the top of all our EAMxx pages?